plenum/server/node.py

```diff
@@ -2441,16 +2487,8 @@ def no_more_catchups_needed(self):
         self._catch_up_start_ts = 0
         if self.ledgerManager.last_caught_up_3PC == (0, 0):
             self.last_sent_pp_store_helper.erase_last_sent_pp_seq_no()
-        self.view_changer.on_catchup_complete()
-        # TODO: need to think of a better way
-        # If the node was not participating but has now found a primary,
-        # then set mode to participating, can happen if a catchup is triggered
-        # without a view change or node start
-        if not self.isParticipating and self.master_replica.hasPrimary:
```

Comment:
don't need it since we will start participate in any any case in selection_primaries_after_catchup